### ISSUE TITLE
F and f don't work

### ISSUE THREAD

**Issue by pklapperich** [jmoon018/PacVim#39]
Expected:
`Fa` should move to the next `a` left while `fa` should move to the next a right. 
Currently:
Neither moves.

**Comment by TamaMcGlinn** [comment]
right, but should it be a jump or a move?

**Comment by pklapperich** [comment]
It's been a long time since I've played this, but I think it should be a move, just like `B`, `b`, `W`, `w, etc. Otherwise it feels inconsistent. Looking back at the README, I see there's a documentation bug in that everything in the README is currently defined as a move, but `$` and `^` are both jumps.

Ex: You're on a sentence, with the cursor on `w`:
 > All _w_ork and no play makes Jack a dull boy

You press `fy`, the cursor moves to `y`. 

> All _work and no play_ makes Jack a dull boy

**Comment by TamaMcGlinn** [comment]
I agree it should be a move, touching all characters in between. Note it would be just as nice to support `t` in the same way (one character less far).

**Comment by TamaMcGlinn** [comment]
I returned to this with [my fork](https://github.com/TamaMcGlinn/PacVim) now, but opted for a jump that cannot cross walls, that feels best.
